src/lib/controllers/tokenize-controller.js

```diff
@@ -32,4 +57,74 @@ export default class TokenizeController {
     })
     return false
   }
+
+  /**
+   * Formats options to the format that would be used by ClientAdapter methods
+   * @param {String} tokenizer
+   * @param {Options} definedLocalOptions
+   * @returns {Object} - name: value for each option
+   */
+  static defineTextTokenizationOptions (tokenizer, definedLocalOptions) {
+    if (!this.tokenizeMethods[tokenizer]) {
+      return
+    }
+
+    let tokenizationOptions = { tokenizer: tokenizer }
+
+    if (this.tokenizeMethods[tokenizer].hasOptions && definedLocalOptions) {
+      tokenizationOptions = Object.assign(tokenizationOptions, definedLocalOptions.formatLabelValueList)
+    }
+
+    return tokenizationOptions
+  }
+
+  /**
+   * Upload default options values for tokenizers (for now only for the alpheiosRemoteTokenizer)
+   * @param {StorageAdapter} storage
+   * @returns {Object} - tokenizerName: { sourceType: Options }
+   */
+  static async uploadOptions (storage) {
+    const resultOptions = {}
+
+    for (let i = 0; i < Object.keys(this.tokenizeMethods).length; i++) {
+      const tokenizeMName = Object.keys(this.tokenizeMethods)[i]
+      const tokenizeM = this.tokenizeMethods[tokenizeMName]
+
+      if (tokenizeM.hasOptions && tokenizeM.uploadOptionsMethod) {
+        resultOptions[tokenizeMName] = await tokenizeM.uploadOptionsMethod(storage, tokenizeM)
+      }
+    }
+    return resultOptions
+  }
+
+  /**
+   * Options upload method for alpheiosRemoteTokenizer tokenizer
+   * @param {StorageAdapter} storage
+   * @returns {Object} - tokenizerName: { sourceType: Options }
+   */
+  static async uploadDefaultRemoteTokenizeOptions (storage) {
+    const adapterTokenizerRes = await ClientAdapters.tokenizationGroup.alpheios({
+      method: 'getConfig',
+      params: {
+        storage
+      }
+    })
+
+    if (adapterTokenizerRes.errors.length > 0) {
+      adapterTokenizerRes.errors.forEach(error => {
+        console.log(error)

```

Comment:
I'm wondering if it's better to use the Logger here

Reply:
Logger is not integrated yet - it would be integrated with #30